src/main/java/cs/f10/t1/nursetraverse/logic/parser/ExportCommandParser.java

```diff
@@ -25,18 +23,20 @@
      */
     public ExportCommand parse(String args) throws ParseException {
         requireNonNull(args);
-        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_FILENAME, PREFIX_INDEXES);
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, CliSyntax.PREFIX_FILENAME,
+                CliSyntax.PREFIX_INDEXES);
```

Comment:
Same issue.

Okay, basically this problem is happening with all the constants imported from `CliSyntax` and `Messages` classes. You can have a look through the rest of the files.